OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,36 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+class CallchainSamplePerfEvent : public PerfEvent {
```

Comment:
Can you move it below `SamplePerfEvent`? It splits `dynamically_sized_perf_event_stack_sample` and `SamplePerfEvent` using it.

Reply:
Sure.